Survey/Sources/Presentation/Modules/Splash/SplashView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  SplashView.swift
+//  Survey
+//
+//  Created by Khanh on 22/07/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+import UIKit
+
+struct SplashView: View {
+
+    private var input: SplashViewModel.Input
+    @ObservedObject var output: SplashViewModel.Output
+
+    private let cancelBag = CancelBag()
+    private let loadTrigger = PassthroughSubject<Void, Never>()
+
+    @State private var image: UIImage? = UIImage(named: "ic_background")
+    @State private var fadeInOut = false
+
+    var body: some View {
+        ZStack {
+            Image("ic_background")
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+                .edgesIgnoringSafeArea(.top)
+
+            Image("ic_logo")
+                .resizable()
+                .frame(width: 200, height: 48, alignment: .center)
+                .onAppear(perform: {
+                    withAnimation(Animation.easeInOut(duration: 1).delay(2)) {
+                        fadeInOut.toggle()
+                    }
+                })
+                .opacity(fadeInOut ? 1 : 0)
```

Comment:
Let's use explicit decimal point to differentiate double and int https://nimblehq.co/compass/development/code-conventions/swift/#general.
```suggestion
            Image("ic_logo")
                .resizable()
                .frame(width: 200.0, height: 48.0, alignment: .center)
                .onAppear(perform: {
                    withAnimation(Animation.easeInOut(duration: 1.0).delay(2.0)) {
                        fadeInOut.toggle()
                    }
                })
                .opacity(fadeInOut ? 1.0 : 0.0)
```